python/minpy/__init__.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
```

Comment:
we can keep minpy as standalone for now. but autograd should by in mxnet since it is now mxnet feature. Calling MX*** CAPI from minpy feels weird.